recipes/cellxgene-census/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  name: cellxgene-census
+  version: 1.16.2
+  python_min: 3.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/cellxgene_census-${{ version }}.tar.gz
+  sha256: c007031b1e42b320a191ff1f20ea4099ea935fe5416dffcd710b0e4a94c28ce1
+  patches:
+    - fix-deps.patch
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
Since cellxgene-census uses setuptools-scm to dynamically generate the package version, I had to use the `SETUPTOOLS_SCM_PRETEND_VERSION` trick to get the version correct. Using the new recipe syntax, I think  `SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}` should work. You can verify with `python -c "import cellxgene_census; print(cellxgene_census.__version__)"`

Reply:
Ah good catch. Thank you!